platform/android/build.gradle.kts

```diff
@@ -0,0 +1,47 @@
+buildscript {
+    apply(from = "${rootDir}/gradle/dependencies.gradle.kts")
+
+    repositories {
+        google()
+        maven { url = uri("https://plugins.gradle.org/m2/") }
+        mavenCentral()
+    }
+
+    dependencies {
+        classpath(libs.androidGradlePlugin)
+        classpath(libs.licensesPlugin)
+        classpath(libs.gradleNexus)
+        classpath(libs.kotlinPlugin)
+    }
+}
+
+plugins {
+    alias(libs.plugins.nexusPublishPlugin)
+    alias(libs.plugins.kotlinter) apply false
+    alias(libs.plugins.kotlinAndroid) apply false
+}
+
+allprojects {
+    repositories {
+        mavenCentral()
+        google()
+    }
+}
+
+subprojects {
+    apply(from = "${rootDir}/gradle/dependencies.gradle.kts")
+}
+
+apply(from = "${rootDir}/gradle/publish-root.gradle.kts")
+
+nexusPublishing {
+    repositories {
+        sonatype {
+            stagingProfileId.set(sonatypeStagingProfileId)
+            username.set(ossrhUsername)
```

Comment:
CI failed because these variables are undefined.

Reply:
where can I find these values @louwers ?

Reply:
Looks like they are in `platform/android/gradle/publish-root.gradle.kts` now

Reply:
fixed it